packages/app/src/app/components/CreateSandbox/Import/SuggestedRepositories.tsx

```diff
@@ -0,0 +1,224 @@
+        <>
+          <StyledList as="ul" direction="vertical" gap={1}>
+            {githubRepos.data?.map(repo => {
+              const importUrl = v2DefaultBranchUrl({
+                owner: repo.owner.login,
+                repoName: repo.name,
+                workspaceId: activeTeamInfo.id,
+                importFlag: true,
+              });
+
+              return (
+                <InteractiveOverlay key={repo.id}>
+                  <StyledItem>
+                    <Stack gap={4} align="center">
+                      <Icon name="repository" color="#999999" />
+                      <InteractiveOverlay.Anchor
+                        href={importUrl}
+                        onClick={() => {
+                          const isPersonalRepository =
+                            repo.owner.login ===
+                            githubAccounts?.data?.personal?.login;
+
+                          if (isPersonalRepository && isTeamSpace) {
+                            track(
+                              'Suggested repos - Imported personal repository into teamspace',
+                              {
+                                codesandbox: 'V1',
+                                event_source: 'UI',
+                              }
+                            );
+                          }
+                        }}
+                      >
+                        <VisuallyHidden>Import</VisuallyHidden>
+                        <Text size={13}>{repo.name}</Text>
+                      </InteractiveOverlay.Anchor>
+                      {repo.private ? (
+                        <>
+                          <VisuallyHidden>Private repository</VisuallyHidden>
+                          <Icon name="lock" color="#999999" />
+                        </>
+                      ) : null}
+                      {repo.updatedAt ? (
+                        <Text size={13} variant="muted">
+                          <VisuallyHidden>Last updated</VisuallyHidden>
+                          {formatDistanceStrict(
+                            zonedTimeToUtc(repo.updatedAt, 'Etc/UTC'),
+                            new Date(),
+                            {
+                              addSuffix: true,
+                            }
+                          )}
+                        </Text>
+                      ) : null}
+                    </Stack>
+                    <StyledIndicator aria-hidden>Import</StyledIndicator>
+                  </StyledItem>
+                </InteractiveOverlay>
+              );
+            })}
+          </StyledList>
+          {restrictsPrivateRepos ? <AuthorizeMessage /> : null}
+        </>
+      ) : null}
+    </Stack>
+  ) : null;
+};
+
+const AuthorizeMessage = () => {
+  const { signInGithubClicked } = useActions();
+
+  return (
+    <Stack gap={2} align="center">
+      <Text
+        variant="muted"
+        size={12}
+      >{`Don't see all your repositories?`}</Text>
```

Comment:
```suggestion
      >Don't see all your repositories?</Text>
```

Reply:
Huh, interesting my VSCode was giving me issues with the `'`. Changed it, thanks!

Reply:
Oh the linter is too, but CSB isn't:
```
173:12  error  `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`  react/no-unescaped-entities
```

Reply:
Fixed it with `&apos;`